meta-phosphor/recipes-phosphor/certificate/phosphor-certificate-manager_git.bb

```diff
@@ -34,3 +35,6 @@ SYSTEMD_SERVICE:${PN} = "${CERT_TMPL}"
 PACKAGECONFIG ??= ""
 PACKAGECONFIG[ibm-hypervisor-cert] = "--enable-ca-cert-extension,,"
 SYSTEMD_SERVICE:${PN} += "${@bb.utils.contains('PACKAGECONFIG', 'ibm-hypervisor-cert', 'bmc-vmi-ca-manager.service', '', d)}"
+
+PACKAGECONFIG[ibm-acf] = "--enable-acf-cert-extension,,"
+SYSTEMD_SERVICE:${PN} += "${@bb.utils.contains('PACKAGECONFIG', 'ibm-acf', 'bmc-acf-manager.service', '', d)}"
```

Comment:
it will add systemd service file to non ibm systems too, so assume this also should go to .bbappend. Do check by compiling non ibm systems, or else CI will fail I guess